aries-core/package.json

```diff
@@ -5,20 +5,20 @@
   "license": "Apache-2.0",
   "main": "src/js/index.js",
   "devDependencies": {
-    "@babel/cli": "^7.7.0",
-    "@babel/core": "^7.1.2",
-    "@babel/node": "^7.7.0",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/cli": "^7.19.3",
+    "@babel/core": "^7.20.5",
+    "@babel/node": "^7.20.5",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
     "babel-loader": "^8.2.5",
-    "del": "^5.1.0",
+    "del": "^6.1.1",
     "dotenv": "^8.2.0",
-    "fs-extra": "^8.1.0",
-    "simple-git": "^3.5.0",
-    "webpack": "^4.44.2",
-    "webpack-cli": "^4.8.0",
-    "webpack-dev-server": "^3.1.14"
+    "fs-extra": "^11.1.0",
+    "simple-git": "^3.15.1",
+    "webpack": "^5.75.0",
```

Comment:
When I run `yarn build` it looks like it is still configured to use webpack 4

Reply:
Yes, I haven't touched any of the `aries-site` configurations yet because it will require some more modifications (including the one you mentioned). This PR only touches things in `aries-core`.

Reply:
Good catch though, and agree we'll need to remove that flag on a subsequent PR.